src/main/java/com/minecolonies/coremod/items/ItemSupplyChestDeployer.java

```diff
@@ -1,6 +1,8 @@
 package com.minecolonies.coremod.items;
 
+import com.minecolonies.api.configuration.Configurations;
 import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.api.configuration.Configurations;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)